src/System.Net.Requests/tests/HttpWebRequestTest_1.7.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Http;
+using System.Net.Test.Common;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Tests
+{
+    public class HttpWebRequestTest17
+    {
+        public readonly static object[][] EchoServers = System.Net.Test.Common.Configuration.Http.EchoServers;
+
+        [Theory, MemberData(nameof(EchoServers))]
+        public void Ctor_VerifyHttpRequestDefaults(Uri remoteServer)
```

Comment:
Please look at the naming convention used in existing System.Net tests.  We use a 3-part name:

```
<method or property under test>_<conditions>_<expectedresult>
```